src/assets/styles/styled.d.ts

```diff
@@ -0,0 +1,6 @@
+import {} from 'styled-components';
```

Comment:
Definicje wszystkich typów globalnych powinny się znajdować w jednym folderze, np `src/types`. Chodzi o to, żeby potem szukając jakiś elementów można było łatwo wiedzieć gdzie to jest, a nie szukać po folderach